package.json

```diff
@@ -30,7 +30,9 @@
   "activationEvents": [
     "onLanguage:dockerfile",
     "onLanguage:yaml",
+    "onCommand:vscode-docker.acrLogs",
     "onCommand:vscode-docker.api.configure",
+    "onCommand:vscode-docker.ACR-queueBuild",
```

Comment:
queue-ACR-build to be consistent with the others (or quick build or whatever you end up calling it)

Reply:
Yes, I agree. 
This change has already been done, in the next PR. I just didn't add it here because to avoid Github messing up more when merging.